release/release_gold.sh

```diff
@@ -68,6 +68,11 @@ function commit_to_branch_and_send_pull_request {
 }
 
 function main {
+	if [[ " ${@} " =~ " --test " ]]
```

Comment:
@kiwm, can you adjust the commit message to be like `Making the release gold ready for testing`?